oauth.rst

```diff
@@ -48,46 +48,20 @@ Next, Configure Netmaker with the following environment variables. If any are le
     SERVER_HTTP_HOST: "api.<netmaker base domain>"
     FRONTEND_URL: "https://dashboard.<netmaker base domain>"
     AZURE_TENANT: "<only for azure, you may optionally specify the tenant for the OAuth>"
-    OIDC_ISSUER: "<only for oidc, your issuer endpoint for OIDC ie. http://127.0.0.1:5556/dex"
+    OIDC_ISSUER: "<only for oidc, your issuer endpoint for OIDC> ie. http://127.0.0.1:5556/dex"
 
 After restarting your server, the Netmaker logs will indicate if the OAuth provider was successfully initialized:
 
 .. code-block::
 
    sudo docker logs netmaker
 
-Once successful, users can click the key symbol on the login page to sign-in with your configured OAuth provider.
+Once successful, users can click on the "Login with SSO" button in the login page to sign-in with your configured OAuth provider.
 
 .. image:: images/oauth1.jpg
    :alt: Login Oauth
    :align: center
 
-
-Configuring User Permissions
```

Comment:
This deleted section should be moved to https://docs.netmaker.io/user-management.html